src/keybase/encode.iced

```diff
@@ -34,15 +35,17 @@ read_base64 = (raw) ->
 
 #=================================================================================
 
-unseal = (buf) ->
+unseal = ({buf, strict}) ->
```

Comment:
`unseal` changed its signature and no tests broke?

I wonder if there is a chance a dependency breaks

Reply:
happy to change it to be backwards-compatible

Reply:
thanks! I was going to say that we can bump major version and it should be easy to check if nothing in our repo uses this; but backwards compatible is fine too.